integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/car/deployment/test/XSLTTransformationCarTestCase.java

```diff
@@ -18,173 +18,36 @@
-            throw new Exception("Service Invocation Failed > " + axisFault.getMessage(), axisFault);
-        }
-        Assert.assertNotNull(response, "Response message null");
-        Assert.assertTrue(response.toString().contains("Code"), "Code element not found in response message");
-        Assert.assertTrue(response.toString().contains("XSLTTransformation"),
-                "Symbol not found on the response message");
-
-    }
-
-    @Test(groups = { "wso2.esb" }, description = "test proxy service invocation", dependsOnMethods = {
-            "artifactDeploymentAndServiceInvocation" })
-    public void deleteCarFileAndArtifactUnDeploymentTest() throws Exception {
-        applicationAdminClient.deleteApplication(carFileName);
-        isCarFileUploaded = false;
-        Assert.assertTrue(isCarFileUnDeployed(carFileName), "Car file undeployment failed");
-        TimeUnit.SECONDS.sleep(5);
-        Assert.assertTrue(esbUtils.isEndpointUnDeployed(context.getContextUrls().getBackEndUrl(), getSessionCookie(),
-                "stockQuoteServiceEndpoint"), "stockQuoteServiceEndpoint Endpoint deployment failed");
-        Assert.assertTrue(esbUtils.isProxyUnDeployed(context.getContextUrls().getBackEndUrl(), getSessionCookie(),
-                "xsltTransformationProxy"), "xsltTransformationProxy Proxy service deployment failed");
-
-        Assert.assertFalse(isResourceExist("/_system/config/transform.xslt"),
-                "transform.xslt deployed by car file not removed from registry");
-        Assert.assertFalse(isResourceExist("/_system/config/transform_back.xslt"),
-                "transform_back.xslt deployed by car file not removed from registry");
-
-    }
-
-    @AfterClass(alwaysRun = true)
-    public void cleanupArtifactsIfExist() throws Exception {
-        if (isCarFileUploaded) {
-            applicationAdminClient.deleteApplication(carFileName);
-        }
-        super.cleanup();
-    }
-
-    private boolean isCarFileDeployed(String carFileName) throws Exception {
 
-        log.info("waiting " + MAX_TIME + " millis for car deployment " + carFileName);
-        boolean isCarFileDeployed = false;
-        Calendar startTime = Calendar.getInstance();
-        long time;
-        while ((time = (Calendar.getInstance().getTimeInMillis() - startTime.getTimeInMillis())) < MAX_TIME) {
-            String[] applicationList = applicationAdminClient.listAllApplications();
-            if (applicationList != null) {
-                if (ArrayUtils.contains(applicationList, carFileName)) {
-                    isCarFileDeployed = true;
-                    log.info("car file deployed in " + time + " mills");
-                    return isCarFileDeployed;
-                }
-            }
+        String carbonHome = System.getProperty(ServerConstants.CARBON_HOME);
+        System.out.println(carbonHome);
+        String sourcePath = carbonHome + File.separator + "server" + File.separator + "registry"
```

Comment:
Shall we move this path upto registry to the new method, since it is going to be same for all the ones which uses it. WDYT

Reply:
Added the requested change. Please review the latest commit.